CONTRIBUTING.md

```diff
@@ -1,13 +1,57 @@
-Any contribution that you make to this repository will
-be under the Apache 2 License, as dictated by that
-[license](http://www.apache.org/licenses/LICENSE-2.0.html):
-
-~~~
-5. Submission of Contributions. Unless You explicitly state otherwise,
-   any Contribution intentionally submitted for inclusion in the Work
-   by You to the Licensor shall be under the terms and conditions of
-   this License, without any additional terms or conditions.
-   Notwithstanding the above, nothing herein shall supersede or modify
-   the terms of any separate license agreement you may have executed
-   with Licensor regarding such Contributions.
-~~~
+# How to Contribute
+
+This project welcomes third-party code via GitHub pull requests.
+
+You are welcome to propose and discuss enhancements using project [issues](https://github.com/IntelRealSense/realsense-ros/issues).
+
+> **Branching Policy**:
+> The `ros2-development` branch is considered stable, at all times.
+> If you plan to propose a patch, please commit into the `ros2-development` branch, or its own feature branch.
+
+In addition, please run `pr_check.sh` under `scripts` directory. This scripts verify compliance with project's standards:
+
+1. Every example / source file must refer to [LICENSE](https://github.com/IntelRealSense/realsense-ros/blob/ros2-development/LICENSE)
+2. Every example / source file must include correct copyright notice
+3. For indentation we are using spaces and not tabs
+4. Line-endings must be Unix and not DOS style
+5. Every API header file must be able to compile as the first included header (no implicit dependencies)
```

Comment:
5. Not relevant IMO, we do not have an API or a check for that

Reply:
Beside that look great!

Reply:
removed